mysql/toddl.go

```diff
@@ -191,3 +192,31 @@ func cvtPrimaryKeys(conv *internal.Conv, srcTable string, srcKeys []schema.Key)
 	}
 	return spKeys
 }
+
+func cvtForeignKeys(conv *internal.Conv, srcTable string, srcKeys []schema.ForeignKey) []ddl.Foreignkey {
+	var spKeys []ddl.Foreignkey
+	for _, key := range srcKeys {
+		spReferTable, err := internal.GetSpannerTable(conv, key.ReferTable)
+		if err != nil {
+			conv.Unexpected(fmt.Sprintf("Can't map foreign key for source table: %s, referenced table: %s", srcTable, key.ReferTable))
+			continue
+		}
+		var spCols, spReferCols []string
+		for i, col := range key.Columns {
```

Comment:
Might be worth a sanity check that key.Columns and key.ReferColumns have same length.
Otherwise could crash when we access key.ReferColumns[i].

Reply:
Generally we are appending both simultaneously, but sure it's worth a check.